### DISCUSSION TITLE
libvips is used as an image processing engine Is there a rust library?

### DISCUSSION THREAD

**Discussion by zhaohuxing** [libvips/libvips#2492] [Q&A] [answered]
Hi, @jcupitt，Are there any plans to support the image processing engine with rust?

**Comment by jcupitt** [comment, marked as answer]
Hi @zhaohuxing,

I like Rust and might try making a binding at some point, but I've no concrete plans at the moment.

A number of people have made Rust bindings already, but I've not looked at any of them in detail and couldn't recommend one over another.

**Reply by zhaohuxing** [reply]
OK, I look forward to it.
